apps/automation_crews/personal_assistant_crew/Readme.md

```diff
@@ -0,0 +1,36 @@
+# start Only Personal Assistant Crew
+
+## Steps to Run
+**Navigate to the Project Directory:**
+```shell
+cd apps/automation_crews/personal_assistant_crew
+```
+Change to the directory where the `setup.sh`, `run_crew.py`, `requirements.txt` files are located. For example:
```

Comment:
prefer writing steps, not descriptions, e.g
setup.sh
run_crew_py
...

Reply:
resolve